platform/linux-generic/odp_schedule_scalable.c

```diff
@@ -67,6 +67,7 @@ typedef struct {
 static sched_global_t *global;
 
 __thread sched_scalable_thread_state_t *sched_ts;
+__thread uint16_t loop_cnt;
```

Comment:
Should be moved inside `sched_scalable_thread_state_t `.

Reply:
hi matiaselo
   it have a very little risk to use uint16.  
  if high priority sched queue always return packets, maybe increase the loop_cnt one 
 or more cycle. when the low priority sched queue is called in the next time, maybe the loop_cnt is equal to the old value , cause scheduler ignore the queue directly . But  the probability is very very small.   if we use uint64, there is not probability to happen the coincidence, for counter loop need many years  .  uint32     
 is  the middle value to reduce the  coincidence probability, uint32 can make sure time of  the coincidence is second level.   which one(uint16 uint32 uint64) do you want to accept from you view  ?

Reply:
Hi,

As you have noted, the probability of this happening is really small. And even if it did, the result would not be a critical failure. So, I think using uint64 would be overkill. Since you are using this code in your application you can better analyze the potential performance differences between uint16 and uint32, I'm good with either option.